motr/cob.c

```diff
@@ -1848,18 +1849,49 @@ M0_INTERNAL int m0__obj_namei_send(struct m0_op_obj *oo)
 	}
 	cr->cr_cob_attr = cob_attr;
 
-	/* Set layout id for CREATE op.*/
+	/* Skip cob lookup if obj.ob_attr.oa_pver is not empty.
+	 * We are assuming here, if pver is not empty that means we
+	 * have received pver from s3 */
```

Comment:
My suggestion is to make the comments application-agnostic and not s3/everything else. applications can be distinguished by their willingness to set the attributes manually to skip cob lookup.

Reply:
Okay, Will update this comment.